### DISCUSSION TITLE
NR01=0 vs NR01=1

### DISCUSSION THREAD

**Discussion by Spit29** [gopro/labs#458] [Q&A]
Hello everyone, I was just wondering if there is a difference between these 2 settings? NR01=0 vs NR01=1 
Does NR01=0 completely disable the noise reduction? Thank you.

**Comment by dnewman-gpsw** [comment]
All wxyz=0 style commands turns off the hack. So NR01=0 is normal noise reduction , =1 is NR disabled

**Reply by Rabii-Hayani** [reply]
M.David 

I went to know wish situation we should active it or desctivate it 

If i run www.gopro.com labs but im not activate it from the QR codes or in the extra .. its is already activited ??? And i should put the NR01=1 to disactivate it or what ??? 
Ot its al ready activate in normal mode as you told like NR01 =0

**Reply by dnewman-gpsw** [reply]
Normal camera, all enhancements are off.  Same as feature=0.

NR01=1 has the enhancement fully enabled.

**Reply by lyricsdude** [reply]
NR01=1 is completely different than NR01=2. I tested it again. Seems 1 is completely off and 2 is noise reduction with lowest setting.

**Reply by Rabii-Hayani** [reply]
I will test this in a diffrent video by increase them .... 1 2 3 4 5 6 7 8 to see any diffrent

**Reply by dnewman-gpsw** [reply]
I wouldn't agree with that description. NR01=1 - denoise reduction is at the lowest possible,  NR01=2 - denoise reduction is 2%, NR01=50 noise reduction at 50% etc.  Effectively NR01=0 is the same as NR01=100.
